src/renderer/appBadge.ts

```diff
@@ -11,17 +11,19 @@ import { Settings } from "./settings";
 
 let GuildReadStateStore: any;
 let NotificationSettingsStore: any;
+let MessageRequestStore: any;
 
 export function setBadge() {
     if (Settings.store.appBadge === false) return;
 
     try {
         const mentionCount = GuildReadStateStore.getTotalMentionCount();
         const pendingRequests = RelationshipStore.getPendingCount();
+        const messageRequests = MessageRequestStore.getMessageRequestsCount();
         const hasUnread = GuildReadStateStore.hasAnyUnread();
         const disableUnreadBadge = NotificationSettingsStore.getDisableUnreadBadge();
 
-        let totalCount = mentionCount + pendingRequests;
+        let totalCount = mentionCount + pendingRequests + messageRequests;
```

Comment:
why? this code is ported exactly from Discord. they don't include this either. we should keep behaviour consistent.

![image](https://github.com/Vencord/Vesktop/assets/45497981/2848a935-a048-485a-b5b9-17487158a0f9)